webview/src/plots/components/comparisonTable/ComparisonTableRow.tsx

```diff
@@ -76,7 +77,11 @@ export const ComparisonTableRow: React.FC<ComparisonTableRowProps> = ({
                 data-testid="row-images"
                 className={cx(styles.cell, { [styles.cellHidden]: !isShown })}
               >
-                <ComparisonTableCell plot={plot} path={path} />
+                {plot.isMulti ? (
```

Comment:
[Q] Could this rely on the length of `imgs`? Could drop isMulti from the type. Could `isMulti` ever be false for multiple images and visa versa?

Reply:
Done!